src/App.css

```diff
@@ -2,6 +2,7 @@
   --bottomSpacing: 140px;
 }
 
-.ClaymateApp .excalidraw .App-bottom-bar {
+.ClaymateApp {
+  height: 100vh;
   bottom: var(--bottomSpacing);
```

Comment:
```suggestion
```

 It's probably a leftover. If it doesn't do anything, let's remove it. I prefer to keep things simple.